src/main/java/uk/gov/hmcts/divorce/systemupdate/event/SystemHandleSupplementaryEvidence.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.divorce.systemupdate.event;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.api.CCDConfig;
+import uk.gov.hmcts.ccd.sdk.api.ConfigBuilder;
+import uk.gov.hmcts.divorce.common.ccd.PageBuilder;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.divorcecase.model.State;
+import uk.gov.hmcts.divorce.divorcecase.model.UserRole;
+
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.CASE_WORKER;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.CASE_WORKER_BSP_SYSTEMUPDATE;
+import static uk.gov.hmcts.divorce.divorcecase.model.access.Permissions.CREATE_READ_UPDATE;
+import static uk.gov.hmcts.divorce.divorcecase.model.access.Permissions.UPDATE_DELETE;
+
+@Component
+public class SystemHandleSupplementaryEvidence implements CCDConfig<CaseData, State, UserRole> {
+
+    public static final String HANDLE_EVIDENCE = "handleEvidence";
```

Comment:
ditto

Reply:
yes this is the event id used by bulk scan